cmd/gaia/testnets/README.md

```diff
@@ -1,228 +1,185 @@
 
-This document explains how to connect to the Testnet of a [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk/) based blockchain. It can be used to connect to the latest Testnet for the Cosmos Hub.
-
-NOTE: We are aware this documentation is sub-par and are actively working to
-improve both the tooling and the documentation to make this as painless as
+Note: We are aware this documentation is sub-par. We are working to
+improve the tooling and the documentation to make this process as painless as
 possible. In the meantime, join the
-[chat](https://riot.im/app/#/room/#cosmos_validators:matrix.org) for technical support. Thanks very
-much for your patience :)
-
-## Software Setup (Manual Installation)
+[Validator Chat](https://riot.im/app/#/room/#cosmos_validators:matrix.org)
+for technical support. Thanks very much for your patience. :)
 
-Follow these instructions to install the Cosmos-SDK and connect to the latest Testnet. This instructions work for both a local machine and a VM in a cloud server.
+## Setting Up a New Node
 
-If you want to run a non-validator full-node, installing the SDK on a Cloud server is a good option. However, if you are want to become a validator for the Hub's `mainnet` you should look at more complex setups, including [Sentry Node Architecture](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#how-can-validators-protect-themselves-from-denial-of-service-attacks), to protect your node from DDOS and ensure high-availability (see the [technical requirements](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#technical-requirements)). You can find more information on validators in our [website](https://cosmos.network/validators), in the [Validator FAQ](https://cosmos.network/resources/validator-faq) and in the [Validator Chat](https://riot.im/app/#/room/#cosmos_validators:matrix.org).
+These instructions are for setting up a brand new full node from scratch. If you ran a full node on a previous testnet, please skip to [Upgrading From Previous Testnet](#upgrading-from-previous-testnet).
 
-### Install [Go](https://golang.org/)
+### Install Go
 
-Install `go` following the [instructions](https://golang.org/doc/install) in the official golang website.
-You will require **Go 1.10+** for this tutorial.
+Install `go` by following the [official docs](https://golang.org/doc/install).
+**Go 1.10+** is required for the Cosmos SDK.
 
-#### Set GOPATH
+### Install Cosmos SDK
 
-First, you will need to set up your `GOPATH`. Make sure that the location `$HOME` is something like `/Users/<username>`, you can corroborate it by typing `echo $HOME` in your terminal.
-
-Go to `$HOME` with the command `cd $HOME` and open the the hidden file `.bashrc` with a code editor and paste the following lines \(or `.bash_profile` if your're using OS X\).
+Next, let's install the testnet's version of the Cosmos SDK. 
 
 ```
-export GOPATH=$HOME/go
-export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+go get github.com/cosmos/cosmos-sdk
```

Comment:
Some users have reported that go doesn't checkout the directory/deletes it b/c of the lack buildable top level go files.

Haven't really reproduced this.

Reply:
I've experienced something similar before. `go get` does checkout the directory and clones all the files, it just gives you a warning that there are no top level go files in that repo.

Reply:
I think we should stop suggesting `go get` ever and just always do the full mkdir and git clone